task-list-rails/app/controllers/tasks_controller.rb

```diff
@@ -6,10 +6,30 @@ def index
   def show
     id = params[:id]
     @task = Task.find(id)
-    @task_info_hash = @task.attributes
   end
 
   def welcome
     redirect_to "/tasks"
   end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    Task.create(task_params[:task])
+    redirect_to "/"
+  end
+
+  def destroy
+    id = params[:id]
+    Task.destroy(id)
```

Comment:
Should `id` be converted to an integer first? Not sure if this is working currently...